examples/checkpoint_FNO_darcy.py

```diff
@@ -98,29 +96,27 @@
               regularizer=False, 
               training_loss=train_loss)
 
-# %%
-# Grab the model from the 15th epoch checkpoint
-checkpointed_trainer = Trainer(model=model, n_epochs=5,
+
+# resume training from checkpoint
+
+trainer = Trainer(model=model, n_epochs=20,
                   device=device,
                   callbacks=[
                     OutputEncoderCallback(output_encoder),
+                    StateCheckpointCallback(save_dir = './new_checkpoints',
+                                            resume_from_dir='./checkpoints/ep_10')
                         ],             
                   wandb_log=False,
                   log_test_interval=3,
                   use_distributed=False,
-                  verbose=True,
-                  checkpoint_to_load='./checkpoints/ep_15.pt')
+                  verbose=True)
 
-# train and evaluate
 trainer.train(train_loader=train_loader,
-              test_loaders=test_loaders,
+              test_loaders={},
               optimizer=optimizer,
               scheduler=scheduler, 
               regularizer=False, 
-              training_loss=train_loss,
-              eval_losses=eval_losses)
-
-
+              training_loss=train_loss)
```

Comment:
Perhaps we should also show how to resume training in this example. e.g. train for 2/3 epochs, save, and then instantiate a new trainer and resume from saved state